### ISSUE TITLE
inotify not available on Mac OS

### ISSUE THREAD

**Issue by icopp** [datashaman/putio-automator#11]
```
Collecting pyinotify==0.9.6 (from putio-automator)
  Using cached pyinotify-0.9.6.tar.gz
    Complete output from command python setup.py egg_info:
    inotify is not available on macosx-10.12-x86_64
```

It wouldn't be as much of a problem, except that the app documentation assumes you'll only use the Docker container via the included commands.

**Comment by datashaman** [comment]
Sorry for the terribly late response, do you know of any alternative python packages that would work with MacOS in the same manner?

**Comment by datashaman** [comment]
this should be resolved in #34
